pro/generate-documentation-qmd/README.md

```diff
@@ -1,45 +1,44 @@
-
-* `repository` - _optional_. Github repository. Used to create links to the source code and to build nextflow commands in the generated documentation. Must use format "owner/repo-name". Will default to the repo where this action is being run for. Ex: `openpipelines-bio/openpipeline`
-
-* `write_index` - _optional_. Whether or not to also create a quarto index file that joins the different pages for the components and workflows. Options are `'true'` or `'false'`.
-
-* `viash_pro_token` - Viash Pro token.
-
-* `tools_version` - _optional_. Release of Viash tools to use. Will use the latest release by default. Ex: `main_build`
-
-* `working_directory` - _optional_. Folder where this action will be executed. Most useful when using
-`actions/checkout` with a different `path:`, as paths to files in the resulting qmd
-are generated relative to the working directory. For example: checking out a repository 
-with `path: "my_project"`, `input_dir: "my_project/src/"` and not using `working_directory`; will
-result in in paths that look like `my_project/src/...`. Using `path: "my_project"`, 
-`input_dir: "./src/"` and `working_directory: "my_project/"` will generate paths like `./src/...`.
+- `output_dir`: Output folder to write to.
+- `viash_pro_token`: Viash Pro token.
+- `project_directory`: *(Optional)* The directory in which the Viash
+  project config (`_viash.yaml`) resides. Most useful when using
+  `actions/checkout` with a different `path:`, as paths to files in the
+  resulting qmd are generated relative to the working directory.
+  Default: `./`.
+- `component_template`: *(Optional)* Jinja template that can be used to
+  customize the resulting qmd files. By default will use the included
+  template.
+- `dest_path`: *(Optional)* Template for output paths. Can contain
+  `{type}`, `{namespace}` and `{name}` wildcards to create a custom
+  folder hierarchy for storing the output for different component.
+  Default: `{type}s/{namespace}/{name}.qmd`.
+- `ref`: *(Optional)* The git tag. Used to create links to the source
+  code and to build nextflow commands in the generated documentation.
+  Default: `${{ github.ref_name }}`.
+- `repository`: *(Optional)* Github repository. Used to create links to
+  the source code and to build nextflow commands in the generated
+  documentation. Must use format “owner/repo-name” will default to the
+  repo where this action is being run for. Default:
+  `${{ github.repository }}`.
+- `write_index`: *(Optional)* Whether or not to also create an index
+  file. Options are ‘true’ or ‘false’. Default: `false`.
+- `tools_version`: *(Optional)* Release of Viash tools to use. Default:
+  `latest`.
+- `src`: *(Optional)* An override for the `--src` parameter in
+  `viash ns list`. Example: `src/`.
+
+### Outputs
 
 ## Examples
 
-```yaml
+``` yaml
```

Comment:
Idk if this space is required :)

Reply:
This is auto-generated by Quarto when using the gfm format.